src/applications/validate-mhv-account/routes.jsx

```diff
@@ -0,0 +1,18 @@
+import ValidateMHVAccount from './containers/ValidateMHVAccount';
+import ErrorMessage from './containers/ErrorMessage';
+import VerifyIdentity from './containers/VerifyIdentity';
+import CreateMHVAccount from './containers/CreateMHVAccount';
+import UpgradeMHVAccount from './containers/UpgradeMHVAccount';
+
+const routes = {
+  childRoutes: [
+    { path: '/', component: ValidateMHVAccount },
+    { path: 'verify', component: VerifyIdentity },
+    { path: 'create-account', component: CreateMHVAccount },
+    { path: 'upgrade-account', component: UpgradeMHVAccount },
+    { path: 'error', component: ErrorMessage },
+    { path: 'error/:errorCode', component: ErrorMessage },
+  ],
```

Comment:
Should there be a parent component here, ensuring that the user is logged-in or that their account meets certain criteria? Currently, all of these pages are publicly-accessible except for the index route.